build/localnet/cluster-manager/main.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/pokt-network/pocket/logger"
+	"github.com/pokt-network/pocket/runtime"
+	"github.com/pokt-network/pocket/runtime/defaults"
+	"github.com/pokt-network/pocket/shared/crypto"
+	"github.com/pokt-network/pocket/shared/k8s"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+const cliPath = "/usr/local/bin/client"
+
+var (
+	rpcUrl string
+	log    = logger.Global.CreateLoggerForModule("cluster-manager")
+)
+
+func init() {
+	rpcUrl = fmt.Sprintf("http://%s:%s", runtime.GetEnv("RPC_HOST", "v1-validator001"), defaults.DefaultRPCPort)
+}
+
+func main() {
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		panic(err.Error())
+	}
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		panic(err.Error())
+	}
+
+	validatorKeysMap, err := k8s.FetchValidatorPrivateKeys(clientset)
+	if err != nil {
+		panic(err)
+	}
+
+	watcher, err := clientset.CoreV1().Services("default").Watch(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		panic(err)
+	}
+
+	for event := range watcher.ResultChan() {
+		service, ok := event.Object.(*v1.Service)
+		if !ok {
+			continue
```

Comment:
https://github.com/pokt-network/pocket/pull/491/files#r1107999277
> My gut is telling me to add a log line on this continue and the one below

I disagree. K8S is VERY verbose. Signal / noise.